contents/blog/10x-engineers-do-user-interviews.md

```diff
@@ -0,0 +1,132 @@
+category: Engineering
+tags:
+  - Guides
+  - Product engineers
+---
+
+To become a 10x engineer, most people think about improving their typing speed, buying more monitors, or using the latest framework where everything is a macro. These hacks focus on speeding up the time from an idea to a rolled-out implementation. You'll get a 1.5x boost at most.
+
+Where you can truly get 10x results is by zooming out and solving for user impact. Here, leverage comes from choosing the right problem, building the right solution, and using the right tools. This can only happen by deeply understanding your users. Enabling you to focus on their "hair-on-fire" problems and build solutions that are intuitive and effective. The best way to do this? Speak directly to your users.
+
+## Tickets are highly lossy
+
+Your users are real people. They have problems to overcome. They work in a team. They have hopes and dreams. But if the only information that you get is the ticket "Add a button to this page" you miss all that context. And you miss the opportunity to build a significantly better product.
+
+Let's discuss an example from PostHog: we were getting requests for handling revenue in PostHog. If taken at face value, we might work on "Adding revenue support to PostHog" - and have added a custom database field and display options for the common currencies.
+
+Instead, we went deeper - asking them why did they want revenue data in PostHog? We learned that if a teammate created the chart it was often unclear what the axes of charts was referring to. This affected charts containing revenue data but also other types of data such as app response times measured in milliseconds. We refined the problem to support custom prefixes and postfixes and built it within a day. And compared to the original problem definition we improved support for much more than just revenue data.
+
+![Custom Prefix](../images/blog/10x-engineers-do-user-interviews/custom-prefix.png)
+
+### Solving the information bottleneck: direct user-engineering interaction
+
+![PM to engineer](../images/blog/10x-engineers-do-user-interviews/pm-to-engineer.png)
+
+![Direct user-Engineer Interaction](../images/blog/10x-engineers-do-user-interviews/direct-interaction.png)
+
+Traditionally, information flows from the user to the product manager (PM) by an interview and then from the PM to the engineer through a bunch of tickets. At each stage, important context is lost. It’s one way, so you can’t go deeper on the important areas.
+
+Direct two-way interaction gives you this context. By significantly increasing the signal you can quickly cut scope, realize you’re tackling the wrong problem, or come up with a more elegant solution.
+
+And it's the engineers who can most effectively cut scope and reframe solutions, as they have the background on the existing architecture and components. PM interviews are still useful to highlight important problem areas and bring in additional context. But they are not a replacement.
```

Comment:
I wonder if it's better to play in more to my engineering side and so it it coming from a peer rather than a PM telling engineers to do more interviews? Could have it a bit more stealthy. Whatever you think will land best?